spec/07-variables.md

```diff
@@ -50,6 +50,10 @@ A constant defined outside of a class or interface is a superglobal
 ([§§](#general)). A constant has static storage duration ([§§](04-basic-concepts.md#storage-duration)) 
 and is a non-modifiable lvalue.
 
+**Undefined Constants**
```

Comment:
I would put it in expressions, rather than here, as it is unclear here what "Undefined constant" is.